### DISCUSSION TITLE
Actual Budget stuck while initializing the database

### DISCUSSION THREAD

**Discussion by wallenium** [tteck/Proxmox#3011] [Help Needed]
Hi there,
need help. Installed ActualBudget with default settings. WebUI is stuck in "Initializing the connection to the local database..."

Anyone any idea why it is failing to initialize the database?

Deleted the container and installed again, no improvement.

Thanks a lot
Simon

**Comment by davemcmath** [comment]
Did you manage to resolve this, I have the same on docker and cannot resolve it...

**Comment by aleduc-cyb** [comment]
Got the same problem here, it is working on Firefox on my PC but fails to load on iPhone (tried Safari, Brave and Firefox). I found this in the docs (https://actualbudget.org/docs/troubleshooting/shared-array-buffer) but the parameters mentioned were already activated.

**Comment by towerhand** [comment]
Make sure you are behind a reverse proxy to access the website via HTTPS or refresh the page and follow this: https://github.com/tteck/Proxmox/discussions/2727
At least, that is how I got it working on my end.

**Reply by Sabin-DC** [reply]
Do I need to do this if im only gonna access on LAN? It works when I use localhost on the PC hosting the server, but if I use my local IP on my phone, i'm also stuck on "initializing the connection to the local database"

**Reply by N2200** [reply]
Yes. I just ran into the same problem, but it only lets you connect with localhost or a https connection. I use nginx proxy manager, but I'm sure there are other ways too
